web-client/public/gene/info.html

```diff
@@ -252,13 +252,13 @@ <h5 class="mb-0">
                   <div class="card-block">
                     <dl class="row">
                       <dt class="col-sm-3">Gene Summary: </dt>
-                      <dd class="geneSummary col-sm-9"></dd>
+                      <dd class="geneSummary col-sm-9 sgdSource"></dd>
                       <dt class="col-sm-3">Molecular Function</dt>
-                      <dd class="molecularFunction col-sm-9"> </dd>
+                      <dd class="molecularFunction col-sm-9 sgdSource"> </dd>
                       <dt class="col-sm-3">Biological Process: </dt>
-                      <dd class="biologicalProcess col-sm-9"></dd>
+                      <dd class="biologicalProcess col-sm-9 sgdSource"></dd>
                       <dt class="col-sm-3">Cellular Component: </dt>
-                      <dd class="cellularComponent col-sm-9"></dd>
+                      <dd class="cellularComponent col-sm-9 sgdSource"></dd>
                     </dl>
                   </div>
                 </div>
```

Comment:
Once the transition is fully tested, this file can go away, yes?

Reply:
Yes. I would feel most comfortable with removing it with the *next* pull request, however, to ensure that the changes work entirely on the `beta` branch.

Reply:
Sounds good; keep it until you are comfortable that it's safe to delete. Just wanted to confirm the intent.